packages/adelus/doc/index_my.doc

```diff
@@ -0,0 +1,59 @@
+/*! \mainpage Trilinos/Pliris: Object-Oriented Interface to a Direct Matrix Solve (Dense matrices)
```

Comment:
Should this file be updated to reflect the name change from Pliris to Adelos?

Reply:
I did not notice this file. Name was changed to Adelus. Thanks, @SAdamBlake.